bluepysnap/frame_report.py

```diff
@@ -106,9 +106,11 @@ def get(self, group=None, t_start=None, t_stop=None):
         if len(view.ids) == 0:
             return pd.DataFrame()
 
-        res = pd.DataFrame(data=view.data,
-                           columns=pd.MultiIndex.from_arrays(np.asarray(view.ids).T),
-                           index=view.times).sort_index(axis=1)
+        res = pd.DataFrame(
+            data=view.data,
+            columns=pd.MultiIndex.from_arrays(np.asarray(view.ids, dtype=np.int64).T),
```

Comment:
should n`p.int64` be `IDS_DTYPE`?

Reply:
From what I see `IDS_DTYPE` is used for nodes and edges ids.
In this case the numpy array used for the columns is an array with shape (N, 2), where the first column represents the node ids and the second the section ids.
Considering that they are still ids, we could use `IDS_DTYPE` also in this case, or call directly `utils.ensure_ids`, what do you think?

Reply:
I think only the first column needs to be set to IDS_DTYPE, so which-ever way makes that cleanest.

I think adding a comment explaining why we're moving from unsigned -> signed is worthwhile though - just a link to the numpy ticket would do, imo.

Reply:
Discussed and agreed to make int64 both cell ids and section ids, to avoid any issue with automatic conversions and for consistency with snap when using libsonata<0.1.10.